wd_demo.au3

```diff
@@ -75,8 +75,8 @@ Func _WD_Demo()
 
 	$iPos += $iSpacing
 	GUICtrlCreateLabel("Update", 15, $iPos + 2)
-	Local $idUpdate = GUICtrlCreateCombo("No", 75, $iPos, 100, 20, $CBS_DROPDOWNLIST)
-	GUICtrlSetData($idUpdate, "32bit|32bit+Force|64Bit|64Bit+Force", "No")
+	Local $idUpdate = GUICtrlCreateCombo("Null - check", 75, $iPos, 100, 20, $CBS_DROPDOWNLIST)
```

Comment:
Not a great description IMO. Would prefer something like "Report only" or "Check; No update".